Python/ceval.c

```diff
@@ -4581,12 +4581,9 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
             DEOPT_IF(self_cls->tp_version_tag != read_u32(cache->type_version),
                      LOAD_METHOD);
             /* Treat index as a signed 16 bit value */
-            int dictoffset = *(int16_t *)&cache->dict_offset;
+            int dictoffset = self_cls->tp_dictoffset;
```

Comment:
I just realised this warns on MSVC. It seems that this `dictoffset` and the one below needs to be `Py_ssize_t`

Reply:
https://github.com/python/cpython/pull/93571